src/mscorlib/src/System/Text/Encoding.cs

```diff
@@ -88,16 +88,18 @@ namespace System.Text
 #endif
     public abstract class Encoding : ICloneable
     {
-        private static volatile Encoding defaultEncoding;
-        private static volatile Encoding unicodeEncoding;
-        private static volatile Encoding bigEndianUnicode;
-        private static volatile Encoding utf7Encoding;
-        private static volatile Encoding utf8Encoding;
-        private static volatile Encoding utf32Encoding;
-        private static volatile Encoding asciiEncoding;
-        private static volatile Encoding latin1Encoding;
+        // We lazily-initialize the static Encoding properties by storing them in
+        // nested static classes. The static constructor, which holds the initialization
+        // logic, will only be run on the first access to a static member of the class
+        // (see dotnet/coreclr#1193 for more info). Thus, we get the benefits of both
```

Comment:
Done. I deleted the comment since it no longer applies (there are no longer any nested static classes). I also removed the issue numbers from the other comments in ASCIIEncoding/etc.